functions/.znap.clone.task

```diff
@@ -10,9 +10,10 @@ private repo=~znap/$url:t:r
 [[ -d $repo ]] &&
     return 1
 
+private -a git_config=()
 is-at-least 2.8 ${${=$( git --version )}[3]} &&
-    private jopt=-j0
-if git -C ~znap clone --depth 1 $jopt --recurse-submodules --shallow-submodules $url; then
+    git_config+=( -c fetch.parallel=0 -c submodule.fetchJobs=0 )
```

Comment:
Please follow the style of the original code. So, inline the `private` declaration.

Reply:
I'm unable to reproduce this with Zsh 5.8.1:
```zsh
% cd $(mktemp -d)
% mkdir -p Git
% > .zshrc <<EOF
PS1='%# ' PS2= RPS2='%^'; setopt transientrprompt interactivecomments
zstyle ':znap:*' repos-dir $PWD/Git
source /Users/marlon/Git/zsh-snap/znap.zsh
% EOF
% env -i HOME=$PWD PATH=$PATH TERM=$TERM ${TERMINFO:+TERMINFO=$TERMINFO} zsh -d 
% znap clone zsh-users/zsh aureliojargas/clitest 
Cloning into 'clitest'...
Cloning into 'zsh'...
remote: Enumerating objects: 92, done.
remote: Counting objects: 100% (92/92), done.
remote: Compressing objects: 100% (63/63), done.
remote: Enumerating objects: 1703, done.
remote: Counting objects: 100% (1703/1703), done.
remote: Total 92 (delta 4), reused 53 (delta 1), pack-reused 0
Receiving objects: 100% (92/92), 44.74 KiB | 1.21 MiB/s, done.
Resolving deltas: 100% (4/4), done.
remote: Compressing objects: 100% (1660/1660), done.
remote: Total 1703 (delta 27), reused 1028 (delta 17), pack-reused 0
Receiving objects: 100% (1703/1703), 4.11 MiB | 10.12 MiB/s, done.
Resolving deltas: 100% (27/27), done.
% znap status
clitest ✓
zsh ✓
% znap pull
%
```

Reply:
Can you share your test environment? I would be interested in knowing what causes that.

Reply:
Thanks, that's interesting. I wonder if it's a problem in `clitest`. I'll have to investigate.

Reply:
Never mind, it was a problem in how I set up the tests. I'll push in a commit soon that fixes this.